api/client/web/README.md

```diff
@@ -0,0 +1,12 @@
+# OpenMeter Web Client
+
```

Comment:
Please add a blurb about when users should use this library. Maybe add a similar to Node SDK too.
Something like:
> Client library for integrating web applications with OpenMeter. Useful to build customer dashboards. The library comes with integrations for React hooks.